pytorch/1. The Torch Tensor Library and Basic Operations.ipynb

```diff
@@ -0,0 +1,585 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## PyTorch Tutorial\n",
+    "MILA, November 2017"
```

Comment:
You can put your name here.
I've seen you have acknowledge sources in other notebooks, that's good :)